### ISSUE TITLE
Book no longer loads?

### ISSUE THREAD

**Issue by Ttibsi** [nathaneastwood/fablore_backup#1]
I was just reading the book and trying to load a new page and it seemed to change url and just throw up a NSURLErrorDomain error for me. I saw the latest commit appears to have changed the URL (and this is a fresh repo too?)

**Comment by nathaneastwood** [comment]
Hi, apologies for this. I was in the process of purchasing a domain. It should all now be available again at https://legendarystories.net

**Comment by nathaneastwood** [comment]
The repo isn't "fresh" per say, but the git history was purged as I had to remove some old png/jpg files that were clogging up the git lfs. It was easier to start afresh than to purge the files.

**Comment by Ttibsi** [comment]
Thank you - apologies for jumping the gun there. Are you part of LegendStudios then? I assumed this was a fan project? (Not that it matters either way - I'm reading this coming from the MTG world as an introduction to the FaB lore, and am enjoying it so far. :) )

**Comment by nathaneastwood** [comment]
No you didn't jump the gun. This was on me, and I'm sorry I interrupted your reading. I am very new to web development (though I'm an experienced R programmer) so I just messed up a little bit at that moment in time not setting the DNS correctly.

I'm in no way affiliated with LSS and this site is entirely fan made :) it's a complete passion project. I wanted an easy way to find the lore and read it in one place. LSS have seen the site and have given me their blessing to continue with it! I'm glad that you are enjoying the lore/site :)
